### PR TITLE
Update on template UI layout

### PR DESCRIPTION
**Pull request by GabrielChouinardLetourneau**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Other... Please describe: **New UI layout**

## Description
**Settings**
image
**Welcome/onboarding**
image
**Dad jokes page**
image

This is a part of the work, more will come (Posts, Welcome, Login)
- Introduce the Dad jokes page instead of Chuck Norris
- Add filters page visuals (non-functional)
- Changes in colors (preliminary to design approval)
- Introduce new layout for the Settings page
- More

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

**Comment by Iguins** [comment]
@MatFillion should we not have a styleguide to be integrated too?

File: src/app/ApplicationTemplate.Shared/Presentation/DadJokes/DadJokesPageViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+using Chinook.DataLoader;
+using Chinook.DynamicMvvm;
+using Chinook.StackNavigation;
+using DynamicData;
+
+namespace ApplicationTemplate.Presentation
+{
+	class DadJokesPageViewModel : ViewModel
+	{
+
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Shared/Presentation/DadJokes/DadJokesPageViewModel.cs:17]: remove space

File: src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesFiltersPage.xaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	<Grid Background="{StaticResource ApplicationPageBackgroundThemeBrush}">
+
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition Height="Auto" />
+		</Grid.RowDefinitions>
+
+		<CommandBar Content="Filters"
+					
+					x:Uid="DadJokesFiltersPage_CommandBar" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesFiltersPage.xaml:29]: Spaces

File: src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:dl="using:Chinook.DataLoader"
+	  xmlns:ue="using:Nventive.View.Extensions"
+	  xmlns:u="using:Nventive.View.Controls"
+	  mc:Ignorable="d">
+
+	<Page.Resources>
+		<DataTemplate x:Key="JokesEmptyTemplate">
+			<Grid>
+
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml:14]: Space

File: src/app/ApplicationTemplate.UWP/Views/Content/Menu.xaml [hunk 4]
```diff
@@ -64,21 +64,71 @@
...
+				</Grid.RowDefinitions>
+
+				<PathIcon Data="M21.99 4.5C21.99 3.4 21.1 2.5 20 2.5H4C2.9 2.5 2 3.4 2 4.5V16.5C2 17.6 2.9 18.5 4 18.5H18L22 22.5L21.99 4.5ZM18 14.5H6V12.5H18V14.5ZM18 11.5H6V9.5H18V11.5ZM18 8.5H6V6.5H18V8.5Z"
+						  HorizontalAlignment="Center"
+						  Foreground="white"
+						  Height="24"
+						  Grid.Row="0" />
+				<TextBlock Text="Posts"
+						   x:Uid="Menu_Posts"
+						   HorizontalAlignment="Center"
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/Menu.xaml:105]: Tabulation

File: src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml [hunk 5]
```diff
@@ -30,69 +31,78 @@
...
+
+				<StackPanel Grid.Row="1"
+							HorizontalAlignment="Center"
+							VerticalAlignment="Bottom">
+
+					<Grid Background="Transparent"
+						  ue:MultipleTapExtension.Command="{Binding NavigateToDiagnosticsPage}"
+						  ue:MultipleTapExtension.TapCount="3">
+						<TextBlock Style="{StaticResource MaterialCaption}">
+							<Run Text="Version"
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml:99]: Tab

**Comment by jeanplevesque** [comment]
@GabrielChouinardLetourneau can you add screenshots to the PR description to give us an idea of the result?

**Comment by jeanplevesque** [comment]
> @GabrielChouinardLetourneau can you add screenshots to the PR description to give us an idea of the result?

@GabrielChouinardLetourneau I meant screenshots of the actual app, not from figma 😅

**Review by Iguins — APPROVED** [review]

File: src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj [hunk 6]
```diff
@@ -93,6 +93,9 @@
     <PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="3.1.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.2" />
+    <PackageReference Include="System.Reactive">
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:96]: put in 1 line

File: src/app/ApplicationTemplate.UWP/Views/Content/Posts/PostsPage.xaml [hunk 7]
```diff
@@ -9,57 +9,75 @@
...
 
 		<Grid.RowDefinitions>
 			<RowDefinition Height="Auto" />
 			<RowDefinition Height="Auto" />
-			<RowDefinition Height="Auto" />
-			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
 
 		<CommandBar Content="Posts"
+					Grid.Row="0"
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/Posts/PostsPage.xaml:20]: Tabulation

**Review by carlh98 — APPROVED** [review]

**Pull request merged**